pkgs/servers/monitoring/prometheus/kthxbye.nix

```diff
@@ -0,0 +1,53 @@
+{ pkgs
+, lib
+, buildGo119Module
+, fetchFromGitHub
+, gnumake
+, go_1_19
+, glibc
+, coreutils
+, nixosTests
+}:
+
+buildGo119Module rec {
+  pname = "kthxbye";
+  version = "0.15";
+
+  src = fetchFromGitHub rec {
+    owner = "prymitive";
+    repo = "kthxbye";
+    rev = "v${version}";
+    hash = "sha256-N1MzutjzLk9MnE1b7dKRsiS7LL4Nb61+NpmjTBPGohI=";
+  };
+
+  vendorHash = "sha256-PtINxblqX/wxJyN42mS+hmwMy0lCd6FcQgmBnxTUdcc=";
+
+  nativeBuildInputs = [
+    gnumake
+    go_1_19
+    glibc
+    coreutils
+  ];
+
+  buildPhase = ''
+    make
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ./kthxbye $out/bin
+    chmod +x $out/bin/kthxbye
+  '';
+
+  passthru.tests = {
+    kthxbye = nixosTests.kthxbye;
+  };
+
+  meta = with lib; {
+    description = "Prometheus Alertmanager alert acknowledgement management daemon";
+    homepage = "https://github.com/prymitive/kthxbye";
+    downloadPage = "https://github.com/prymitive/kthxbye/releases";
```

Comment:
```suggestion
```
thats only interesting when the download link is not obvious which is mostly the case for custom websites and proprietary stuff